examples/esm/index.mjs

```diff
@@ -78,4 +78,15 @@ import fs from 'fs';
     signal: AbortSignal.timeout(10_000), // cancel the upload after 10 seconds
   });
   console.log(JSON.stringify(uploadResult, null, 2));
+
+  /**
+   * Tops up a wallet with Credits using tokens.
+   * Default token is AR, using Winston as the unit.
+   */
+  const topUpResult = await turboAuthClient
+    .topUpWithTokens({
+      tokenAmount: 1, /// 0.000_000_000_000_001 AR
```

Comment:
Could be nice if we had some conversion classes that speak in the native units, e.g.
```
const topUpResult = await turboAuthClient
    .topUpWithTokens(WinstonToTokenAmount(1));
```
So you'd have functions like:
`ARtoTokenAmount`
`SOLtoTokenAmount`
`LamportsToTokenAmount`
`EtherToTokenAmount`
`WeiToTokenAmount`
Is this useful? I'm not sure if we're handling each of these in their smallest whole units.

Reply:
I'm just worried people will specify `tokenAmount` incorrectly and get really pissed off or confused when they try to send up 1AR but send 1Winston instead 😳

Reply:
its worth it probably. still give users the option to provide a number, but our examples can clearly show these utils

Reply:
added here: 37417dd